Fixed #15727 -- Added Content Security Policy (CSP) support. (#18215)
django/django: docs/releases/6.0.txt

What's new in Django 6.0
========================

> Content Security Policy (CSP) Support

Reviewer: We don't title-case headings
```suggestion
Content Security Policy (CSP) support
```

Reviewer: (This comment applies to all titles in the updated docs here.)

-------------------------------------

> Django 6.0 introduces built-in :ref:`Content Security Policy (CSP)
> <security-csp>` support, making it easier to protect your application against
> various types of content injection attacks. CSP helps you specify which content
> sources are allowed to be loaded by the browser.

Reviewer: ```suggestion
The new built-in :ref:`Content Security Policy (CSP) <security-csp>` support
makes it easier to protect your application against content injection attacks.
CSP is a security standard that allows you to specify which resources browsers
are allowed to load.
```

We don't normally say "Django <VERSION>" throughout the release notes. Also, I edited for brevity clarity.

To enable CSP, add the
:class:`~django.middleware.csp.ContentSecurityPolicyMiddleware` to your
settings::

  MIDDLEWARE = [
      # ...
      "django.middleware.csp.ContentSecurityPolicyMiddleware",
      # ...
  ]

You can customize your CSP policy through Django settings:

* :setting:`SECURE_CSP` for the enforced policy
* :setting:`SECURE_CSP_REPORT_ONLY` for the report-only policy

> Here's an an example of a basic CSP config::

Reviewer: ```suggestion
Here's an an example of a basic CSP configuration::
```

Use full words wherever possible.
